apps/nestjs-backend/src/features/import/open-api/import-open-api.service.ts

```diff
@@ -86,6 +92,22 @@ export class ImportOpenApiService {
             } catch (e) {
               this.logger.error((e as Error)?.message, 'Import: Records');
```

Comment:
this.logger.error((e as Error)?.message, (e as Error)?.stack);

this way can track origin stack